go/private/common.bzl

```diff
@@ -12,12 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//go/private:skylib/lib/dicts.bzl", "dicts")
-load("//go/private:skylib/lib/paths.bzl", "paths")
-load("//go/private:skylib/lib/sets.bzl", "sets")
-load("//go/private:skylib/lib/shell.bzl", "shell")
-load("//go/private:skylib/lib/structs.bzl", "structs")
-load("@io_bazel_rules_go//go/private:mode.bzl", "mode_string")
+load("//go/private:skylib/lib/dicts.bzl", _dicts = "dicts")
```

Comment:
There shouldn't be any need to re-export Skylib symbols from common.bzl. .bzl files that need Skylib symbols should load them directly.

Same for `mode_string`, actually.